packages/perspective-jupyterlab/README.md

```diff
@@ -18,3 +19,19 @@ jupyter labextension install @finos/perspective-jupyterlab
 ```bash
 pip install perspective-python
 ```
+
+## Build
```

Comment:
this is not exactly suited to be distributed with the package README, can move this to a DEVELOPMENT.md or similar